MatrixSDKTests/MXBaseKeyBackupTests.m

```diff
@@ -628,21 +607,18 @@ - (void)testEncryptAndDecryptKeyBackupData
 
         // - Pick a megolm key
         MXOlmInboundGroupSession *session = [aliceSession.crypto.store inboundGroupSessionsToBackup:1].firstObject;
+        session.untrusted = self.isUntrusted;
```

Comment:
Why do we need that?
I would suggest to add a sanity check instead to make sure that all sessions handled outside of the key backup are trusted.

```
XCTAssertTrue(session.untrusted);
```

Reply:
Yeah good idea, have added that. But we still need to alter the flag to have the same untrusted value when comparing `sessionData` objects later.